README.md

```diff
@@ -20,78 +20,77 @@ The RHTPA Ansible collection deploys the following RHTPA components:
 
-* 24 vCPU, 
-* 48 GB Ram, 
-* 100 GB Disk space 
+* 24 vCPU 
+* 48 GB RAM
+* 100 GB of fre disk space 
 
 ## Requirements
 
-* Ansible 2.16.0 or greater
-* Python 3.10.0 or greater
-* RHEL x86\_64 9.3 or greater.
+* Ansible 2.16.0 or greater.
+* Python 3.10.0 or greater.
+* Red Hat Enterprise Linux 9.3 or greater for the x86_64 architecture.
 * Installation and configuration of Ansible on a control node to perform the automation.
+* External services:
+    * An OpenID Connect (OIDC) provider.
+    * Simple Queue Service (SQS).
+    * A new PostgreSQL database.
+    * AWS Simple Storage Service (S3) or an S3-compatible service, for example, MinIO.
 
-You must supply the following external services:
+## External services
 
-* An OpenID Connect (OIDC) provider, such [RedHat Single Sign On](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/) or Amazon Web Services (AWS) Cognito.
-* Simple Queue Service (SQS), for example, [Red Hat AMQ Streams](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/amq_streams/)
-* A new PostgreSQL database.
-* AWS Simple Storage Service (S3) or an S3-compatible service, for example, MinIO.
+### OIDC providers
 
-## External Services Configurations
+* [Trustification Keycloak](https://github.com/trustification/trustification/blob/release/1.2.z/docs/modules/admin/pages/cluster-preparing.adoc#keycloak)
+* [Red Hat Single Sign-On](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/)
+* [Amazon Web Services (AWS) Cognito](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-cognito-user-pools.html)
 
-### RedHat Single Sign On
+### SQS
 
-* [Trustification Keycloak](https://github.com/trustification/trustification/blob/release/1.2.z/docs/modules/admin/pages/cluster-preparing.adoc#keycloak)
+* [Trustification event queues](https://github.com/trustification/trustification/blob/release/1.2.z/docs/modules/admin/pages/cluster-preparing.adoc#event-queues)
+* [Red Hat AMQ Streams](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/amq_streams/).
 
+Create the following topic names before installing RHTPA:
 
-### RedHat Kafka streams  
-  With the following topic names created:
 ```
-  bombastic-failed-default
-  bombastic-indexed-default
-  bombastic-stored-default
-  vexination-failed-default
-  vexination-indexed-default
-  vexination-stored-default
-  v11y-failed-default
-  v11y-indexed-default
-  v11y-stored-default
+bombastic-failed-default
+bombastic-indexed-default
+bombastic-stored-default
+vexination-failed-default
+vexination-indexed-default
+vexination-stored-default
+v11y-failed-default
+v11y-indexed-default
+v11y-stored-default
 ```
 
 Configure these topic names in the `roles/tpa_single_node/vars/main.yml` file.
 
-* [Trustification event queues](https://github.com/trustification/trustification/blob/release/1.2.z/docs/modules/admin/pages/cluster-preparing.adoc#event-queues)
```

Comment:
Why this is removed ?

Reply:
It wasn't removed, it just moved to line 51.